src/index.js

```
@@ -72,117 +75,6 @@ require('signal-exit/signals').forEach((signal) => {
   });
 });
 
-// Simplified support only for yml
-const getServerlessFile = (dir) => {
-  const ymlFilePath = path.join(dir, 'serverless-compose.yml');
-  const yamlFilePath = path.join(dir, 'serverless-compose.yaml');
-
-  if (utils.fileExistsSync(ymlFilePath)) {
-    return utils.readFileSync(ymlFilePath);
-  }
-  if (utils.fileExistsSync(yamlFilePath)) {
-    return utils.readFileSync(yamlFilePath);
-  }
-
-  return false;
-};
-
-const isComponentsTemplate = (serverlessFile) => {
```

moved and renamed to `validateConfiguration` (it now throws an exception directly instead of returning a bool)